package-builder/build-manifest.nix

```diff
@@ -91,6 +93,10 @@ pkgs.runCommandNoCC name
           # is instantiated.
           source $stdenv/setup # is this necessary?
 
+          # Set HOME to a _writable_ directory in the build sandbox.
+          # <https://github.com/flox/flox/issues/2092>
+          export HOME="$PWD"
```

Comment:
I couldn't tell "which" `$PWD` the original issue(s) referred to.
I.e. the `$PWD` of the nix build scope or the `$PWD` of the flox build scope.
Or in other words whether it is expected that $HOME is in the same directory as the user source files, or just another writable location.

Similarly, if HOME is not in the source directory, does it matter whether it is the parent directory (incidentally) or should it be a separate branch to avoid directory naming conflicts with the source directory.

```
$HOME
L $name

# VS
.
L $HOME/
L $name/

# VS 

.
L $name == $HOME
```

Reply:
> or just another writable location.

It was this for #2092; `go` needs to create/write the path derived from `$GOCACHE` but it doesn't care where it is.

> Similarly, if HOME is not in the source directory, does it matter whether it is the parent directory (incidentally) or should it be a separate branch to avoid directory naming conflicts with the source directory.

It would be nice-to-have but I don't know where we'd put it and my guess is that other tools that depend on `$HOME` implicitly are going to be careful enough with their naming (e.g. XDG style directories) that conflicts with the source directory are unlikely.